mol_dqn/chemgraph/dqn/run_dqn.py

```diff
@@ -39,12 +39,11 @@
 from rdkit.Chem import QED
 
 import tensorflow as tf
-from tensorflow import gfile
 
-from mol_dqn.chemgraph.mcts import deep_q_networks
-from mol_dqn.chemgraph.mcts import molecules as molecules_mdp
-from mol_dqn.chemgraph.py import molecules
-from mol_dqn.chemgraph.tensorflow import core
+from . import deep_q_networks
+from . import molecules as molecules_mdp
+from .py import molecules
+from .tensorflow_core import core
```

Comment:
Please be consistent with your imports. Here you use `.tensorflow_core`; elsewhere you use just `tensorflow_core` (same for `.py` vs. `py`).

Reply:
Done